content/docs/rosa/gpu/_index.md

```diff
@@ -156,9 +154,10 @@ Two options: [Helm](#helm) or [Manual](#manually)
 1. Wait until NFD instances are ready
 
    ```bash
-   oc wait --for=jsonpath='{.status.numberReady}'=3 -l app=nfd-master ds -n openshift-nfd
+   oc wait --for=jsonpath='{.status.availableReplicas}'=3 -l app=nfd-master deployment -n openshift-nfd
    ```
-   
+   > NOTE: If you are deploying ROSA in single-AZ change the replicas from 3 to 1 nfd-master
```

Comment:
Can we move the note up before we show the commands to run?

Reply:
done!